modules/exploits/linux/http/vmware_nsxmgr_xstream_rce_cve_2021_39144.rb

```diff
@@ -0,0 +1,143 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'securerandom'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'VMware NSX Manager XStream unauthenticated RCE',
+        'Description' => %q{
+          VMware Cloud Foundation (NSX-V) contains a remote code execution vulnerability via XStream open source library.
+          VMware has evaluated the severity of this issue to be in the Critical severity range with a maximum CVSSv3 base score of 9.8.
+          Due to an unauthenticated endpoint that leverages XStream for input serialization in VMware Cloud Foundation (NSX-V),
+          a malicious actor can get remote code execution in the context of 'root' on the appliance.
+          VMware Cloud Foundation 3.x and more specific NSX Manager Data Center for vSphere up to and including version 6.4.13
+          are vulnerable to Remote Command Injection.
+
+          This module exploits the vulnerability to upload and execute payloads gaining root privileges.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'h00die-gr3y', # metasploit module author
+          'Sina Kheirkhah, Source Incite', # Security researcher
+          'Steven Seeley,  Source Incite' # Security researcher
```

Comment:
Should just be the names so it's identical to their other entries.
```suggestion
          'Sina Kheirkhah', # Security researcher (Source Incite)
          'Steven Seeley' # Security researcher (Source Incite)
```